src/main/java/duke/helper/Storage.java

```diff
@@ -0,0 +1,74 @@
+package duke.helper;
+
+import duke.tasks.Task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents a storage object which deals with loading, saving the task into the file.
+ *
+ */
+public class Storage {
+    private File file;
+
+
+    public Storage(String filePath) {
+        String[] pathArr = filePath.split("/");
+        String directoryPath = "/" + pathArr[0];
+        try {
+            file = new File(System.getProperty("user.dir") + "/" + filePath);
+            if (!file.isFile()) {
+                File dir;
+                dir = new File(System.getProperty("user.dir") + directoryPath);
+                dir.mkdir();
+                file.createNewFile();
```

Comment:
I like this simple and straightforward way of creating directories and file(s). I would like to suggest using the newer Java NIO Paths and Files library to do something similar such as:
```
 Files.createDirectories(directoryPath);
 Files.createFile(filePath);
```